ecommerce/certprep/static/templates/_offer_course_list.html

```diff
@@ -0,0 +1,113 @@
+<section class="col-xs-12 col-md-8 <% if (courses.length == 1) { %>col-lg-8 col-lg-offset-2<% }else { %>col-lg-12<% } %>">
```

Comment:
We cannot verify the changes you have made to this file, please separate the original template from your changes. @sergivalero20

Reply:
I removed this file because I determined that it wasn't necessary.